src/main/java/lol/fairplay/ghidraapple/analysis/objcclasses/OCStructureAnalyzer.kt

```diff
@@ -0,0 +1,53 @@
+package lol.fairplay.ghidraapple.analysis.objcclasses
+
+import ghidra.app.services.AbstractAnalyzer
+import ghidra.app.services.AnalysisPriority
+import ghidra.app.services.AnalyzerType
+import ghidra.app.util.importer.MessageLog
+import ghidra.program.model.address.AddressSetView
+import ghidra.program.model.data.*
+import ghidra.program.model.listing.Program
+import ghidra.util.task.TaskMonitor
+
+
+class OCStructureAnalyzer : AbstractAnalyzer(NAME, DESCRIPTION, AnalyzerType.BYTE_ANALYZER) {
+
+    companion object {
+        private const val NAME = "Objective-C Structures"
+        private const val DESCRIPTION = ""
+        private val PRIORITY = AnalysisPriority.BLOCK_ANALYSIS.after()
+    }
+
+    init {
+        priority = PRIORITY
+        setPrototype()
+    }
+
+    override fun added(program: Program, set: AddressSetView?, monitor: TaskMonitor?, log: MessageLog?): Boolean {
+        val category = CategoryPath("/GA_OBJC")
+
+        val objc_class_path = arrayOf("objc", "class_t")
+
+        program.symbolTable.symbolIterator.filter {
+            val slice = it.path.slice(0..1)
+
+            return@filter slice.containsAll(objc_class_path.toList())
+                    || it.name.startsWith("_OBJC_CLASS_\$_")
```

Comment:
Is the `return@filter` really needed here? Seems like this should be equivalent to just leave that out, the last expression is implicitly the return value of the lambda

Reply:
I forgot to remove this. I was experiencing some issues earlier and I did this to ensure I wasn't insane.